packages/zoe/test/unitTests/zcf/test-zcf.js

```diff
@@ -194,3 +188,202 @@ test(`zcf.getAmountMath - from issuerKeywordRecord & zcf.saveIssuer`, async t =>
   await zcf.saveIssuer(bucksKit.issuer, 'C');
   compareAmountMath(t, zcf.getAmountMath(bucksKit.brand), bucksKit.amountMath);
 });
+
+test(`zcf.assertUniqueKeyword`, async t => {
+  const { moolaKit, simoleanKit } = setup();
+  const issuerKeywordRecord = { A: moolaKit.issuer, B: simoleanKit.issuer };
+  const { zcf } = await setupZCFTest(issuerKeywordRecord);
+  t.throws(() => zcf.assertUniqueKeyword('A'), {
+    message: 'keyword (a string) must be unique\nSee console for error data.',
+  });
+  t.throws(() => zcf.assertUniqueKeyword('B'), {
+    message: 'keyword (a string) must be unique\nSee console for error data.',
+  });
+  // Unique, but not a valid Keyword
+  t.throws(() => zcf.assertUniqueKeyword('a'), {
+    message:
+      'keyword "a" must be ascii and must start with a capital letter.\nSee console for error data.',
+  });
+  t.throws(() => zcf.assertUniqueKeyword('3'), {
+    message:
+      'keyword "3" must be ascii and must start with a capital letter.\nSee console for error data.',
+  });
+});
```

Comment:
It's normal to include at least one positive test. Any reason not to here?

Reply:
Yup, good point